src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -321,6 +322,19 @@ impl LocalStateStore for LocalHummockStorage {
         .await
     }
 
+    async fn try_flush(&mut self) -> StorageResult<()> {
+        if self.mem_table.kv_size.size() > MEM_TABLE_SPILL_THRESHOLD {
+            tracing::info!(
+                "The size of mem table exceeds 64 Mb and spill occurs. table_id {}",
+                self.table_id.table_id()
+            );
```

Comment:
It can be better to replace `64 Mb` with `{} Mb` and pass in the size threshold with `MEM_TABLE_SPILL_THRESHOLD >> 20`.